app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -439,4 +440,20 @@ private void startScreenService() {
                 Context.MEDIA_PROJECTION_SERVICE);
         startActivityForResult(projectionManager.createScreenCaptureIntent(), SCREEN_CAPTURE_REQ_CODE);
     }
+
+    /**
+     * Handles custom action intents probably from notification.
+     */
+    @Override
+    public void onNewIntent(Intent intent) {
```

Comment:
Noticed this doest fire when starting a new instance of the activity (when GoIV settings screen is killed)